common/testing.cpp

```diff
@@ -125,6 +125,16 @@ ::testing::AssertionResult FilesEqual(const string &filename1, const string &fil
 	return ::testing::AssertionFailure() << filename1 << " and " << filename2 << " differ";
 }
 
+::testing::AssertionResult FilesNotEqual(const string &filename1, const string &filename2) {
```

Comment:
Or assert or whatever

Reply:
Because of the verbosity of `FilesEqual` when the assertion is false. It prints the output on `ls` and the `diff` and it is very confusing to print that when you actually expect it to be different.